packages/eslint-plugin-pf-codemods/src/rules/helpers/nodeMatches/checkMatchingImportDeclaration.ts

```diff
@@ -0,0 +1,30 @@
+import { ImportDeclaration, ImportSpecifier } from "estree-jsx";
+import { pfPackageMatches } from "../pfPackageMatches";
+
+function findSpecifier(
+  node: ImportDeclaration,
+  imporSpecifier: ImportSpecifier
+) {
+  return node.specifiers.find(
+    (specifier) =>
+      specifier.type === "ImportSpecifier" &&
+      specifier.imported.name === imporSpecifier.imported.name
+  );
+}
+
+/** Used to check whether the current ImportDeclaration node matches at least 1 of the import specifiers. */
+export function checkMatchingImportDeclaration(
```

Comment:
Can you refactor this helper to return only boolean? Or if there is a purpose of returning the specifier to return it `as ImportSpecifier` with the type casting. Depends on whether we need the specifier in a return value, but in the array case it would just return the first specifier, so maybe using `.some` everywhere (instead of `.find`) would be better.